plugins/aea-ledger-ethereum/aea_ledger_ethereum/ethereum.py

```diff
@@ -791,6 +792,15 @@ def __init__(self, **kwargs: Any):
             "gas_price_strategies", DEFAULT_GAS_PRICE_STRATEGIES
         )
 
+        if kwargs.pop("poa_chain", False):
+            # https://web3py.readthedocs.io/en/stable/middleware.html#geth-style-proof-of-authority
+            self._api.middleware_onion.inject(
+                geth_poa_middleware, name="geth_poa_middleware", layer=0
+            )
+            _default_logger.info(
+                "EthereumApi has been configured with Proof of Authority chain support"
+            )
+
```

Comment:
Can we also store this variable as a class variable?, we might require this later somewhere.